.github/workflows/.test.yaml

```diff
@@ -21,4 +21,4 @@ jobs:
       - name: Gradle test and build
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./gradlew :apps:${{ github.workflow }}:test --stacktrace
```

Comment:
ref https://docs.gradle.org/current/userguide/configuration_cache.html 
burde vi nok fjerne caching av build filene i stedet for `configuration-cache`?

Reply:
Eller kanskje begge to :D